internal/featuresupport/features_olm_operators.go

```diff
@@ -84,6 +84,7 @@ func (feature *LvmFeature) getIncompatibleFeatures(OCPVersion string) *[]models.
 		models.FeatureSupportLevelIDNUTANIXINTEGRATION,
 		models.FeatureSupportLevelIDVSPHEREINTEGRATION,
 		models.FeatureSupportLevelIDODF,
+		models.FeatureSupportLevelIDOPENSHIFTAI,
```

Comment:
Should we put a comment - perhaps to an issue or otherwise providing some context on why the LVM and SNO features are incompatible with Openshift AI. What causes the problem?

Reply:
@paul-maidment OpenShift AI itself isn't explicitly incompatible with LVM or SNO. But it requires at least two worker nodes (I believe that is a soft limitation that will be soon lifted) and some kind of storage for the most basic things, like creating a workbench or deploying a model. We decided to include ODF in the bundle because that provides block storage (used for workbenches) and also S3-like storage (used to deploy models).

LVM would also work out of the box for workbenches, but doesn't provide directly the S3-like storage: it would require also Nooba.

ODF is incompatible itself with LVM, and that is not new with OpenShift AI: it has been like that since the ODF operator was introduced. I am not sure about the reasons for that, but I guess is related to the fact that with the default configuration both will try to use the same disks for storage, and the result will be unpredictable.

I have the feeling that explaining all these details doesn't belong in this patch, but I will add them if you think it is really needed.

Reply:
No, I think it's OK, if there is a link to a discussion of the issue, maybe that would add context for those who want to look deeper into this?